sqlite-android/src/main/jni/sqlite/android_database_SQLiteConnection.cpp

```diff
@@ -534,13 +534,20 @@ static void nativeResetStatementAndClearBindings(JNIEnv* env, jclass clazz, jlon
 }
 
 static int executeNonQuery(JNIEnv* env, SQLiteConnection* connection, sqlite3_stmt* statement) {
+    int err;
```

Comment:
Whats the main reason for this change ? the original code seems to match whats in the aosp code https://cs.android.com/android/platform/superproject/+/android-11.0.0_r3:frameworks/base/core/jni/android_database_SQLiteConnection.cpp;l=521

Reply:
Hmmm, I merged commit from https://www.sqlite.org/android/info/dfed6feb2b1a71eb but not check in AOSP.
Maybe it was wrong decision, I will rollback this.